runtime/lib/app/ext-app.js

```diff
@@ -6,9 +6,13 @@ var childProcess = require('child_process')
 var path = require('path')
 var logger = require('logger')('ext-app')
 var _ = require('@yoda/util')._
-var ActivityDescriptor = require('./activity-descriptor').ActivityDescriptor
 
-var entry = path.join(__dirname, '..', '..', 'client', 'ext-app-entry.js')
+var kAppModesTest = require('../../constants').AppScheduler.modes.test
+var ActivityDescriptor = require('../descriptor/activity-descriptor')
+var ActivityTestDescriptor = require('../descriptor/activity-test-descriptor')
```

Comment:
About the paths of descriptors, I think we don't need the postfix `descriptor` if we put them in a directory `./descriptor` :)

Reply:
files naming are mostly inspired by way of foo-controller, bar-controller. So the suffix of components are preserved.

Reply:
Where are the *-controller?

Reply:
I mean the convention of naming a controller.